File: src/main/java/me/ikevoodoo/lifestealsmpplugin/commands/VersionCommand.java
package me.ikevoodoo.lifestealsmpplugin.commands;

import me.ikevoodoo.lifestealsmpplugin.LifestealSmpPlugin;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class VersionCommand implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        sender.sendMessage(LifestealSmpPlugin.prefix + ChatColor.GOLD + "Current version: " + ChatColor.GOLD + (LifestealSmpPlugin.updateAvailable ?
                LifestealSmpPlugin.getInstance().getDescription().getVersion() + ChatColor.RED + " (OUTDATED)" : LifestealSmpPlugin.getInstance().getDescription().getVersion() + ChatColor.GREEN + " (LATEST)"));
        return true;
    }
}

File: src/main/kotlin/me/ikevoodoo/lssmp/commands/VersionCommand.kt
package me.ikevoodoo.lssmp.commands

import UpdateData
import me.ikevoodoo.lssmp.LSSMP
import me.ikevoodoo.lssmp.TranslationManager.Companion.getTranslation
import me.ikevoodoo.lssmp.utils.ExtensionUtils.Companion.color
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

class VersionCommand: CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
        if(!sender.hasPermission("lssmp.version")) {
            sender.getTranslation("commands.version.no-perms")?.let { sender.sendMessage(it) }
            return true
        }

        if(UpdateData.UPDATE_AVAILABLE) {
            sender.sendMessage(
                LSSMP.PREFIX + sender.getTranslation("commands.version.message")
                    ?.replace("%version-color%", if (UpdateData.UPDATE_AVAILABLE) "&a" else "&c")
                    ?.replace("%version%",
                        (if (UpdateData.UPDATE_AVAILABLE) UpdateData.VERSION else LSSMP.INSTANCE.description.version)
                            ?: "Unknown"
                    )
                    ?.replace(
                        "%status%",
                        if (UpdateData.UPDATE_AVAILABLE) "(${sender.getTranslation("commands.version.status.outdated")})"
                        else "(${sender.getTranslation("commands.version.status.latest")})"
                    )?.color()
            )

        }
        return true;
    }
}